### PR TITLE
Add Collective APIs Site on Alternative Sites

### PR DESCRIPTION
**Pull request by hyamero**
issuebadge
### [Collective APIs](https://www.collective-api.ml) is a website made using the [Public APIs](https://github.com/public-apis/public-apis) API
`website:` https://www.collective-api.ml/

#### Landing Page
collective-apis

#### Categories
collective-apis-categories

#### API Cards
collective-apis-card

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@
 <div align="center">
     <sub>Alternative sites for the project (unofficials)</sub>
     <br />
+    <a href="https://www.collective-api.ml">Collective APIs</a> •
```
Comment (matheusfelipeog) [inline README.md:46]: You must add this link after Apihouse.

Ex:

```markdown
(...)
    <a href="https://devresourc.es/tools-and-utilities/public-apis">Dev Resources</a> •
    <a href="https://www.public-apis.ml">Public APIs Site</a> •
    <a href="https://apihouse.vercel.app">Apihouse</a> •
    <a href="https://www.collective-api.ml">Collective APIs</a>
</div>
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Wow @hyamero, BIG THANKS for your contribution and for creating this amazing project using Public APIs, we appreciate it. ❤️ 

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (hyamero) [reply README.md]: Hey @matheusfelipeog, thanks for reviewing my project. I have resolved your suggestion: https://github.com/public-apis/public-apis/pull/2068/commits/3f0a7ee29a502354ba6f99d3b648160df8056839 ❤

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for the update ;)

**Pull request merged**